openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMUtils.java

```diff
@@ -46,6 +46,11 @@ public class LightGBMUtils {
      */
     static final int BINARY_LGBM_NUM_CLASSES = 1;
 
+    /**
+     * Environment variable with the implementation of libc available on the system.
+     */
+    static final String FDZ_OPENML_JAVA_LIBC = "FDZ_OPENML_JAVA_LIBC";
```

Comment:
If you aren't accessing this externally, make it private

Reply:
You are right! I changed it to private.